tests/cli/init/init.test.ts

```diff
@@ -51,7 +51,7 @@ test("init command installs @types/react and passes type-checking", async () =>
   expect(tsconfigExists).toBeTrue()
 
   try {
-    const typeCheckResult = execSync("npx tsc --noEmit", {
+    const typeCheckResult = execSync("npx --yes tsc --noEmit", {
```

Comment:
hmm idk about this, i think we should add `typescript` as a devDependency when we do `tsci init`